src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+        analyzed_direct_dependencies_count:
+          type: integer
+          example: 25              
+        transitive_count:
+          type: integer
+          default: -1
+          example: 11
+        vulnerable_transitives_count:
+          type: integer
+          default: -1
+          example: 1
+        license_analysis:
+          $ref: '#/components/schemas/LicenseAnalysis'
+        distinct_licenses:
+          type: array
+          items:
+            type: string
+        stack_license_conflict:
+          type: boolean
+        dependencies:
+          type: array
+          items:
+            $ref: '#/components/schemas/PackageWithTransitives'
+        total_licenses:
+          type: integer
+          example: 15
+        recommendation_ready:
+          type: boolean
+      description: Stack analyses result data
+
+    StackAggregatorResultForRegisteredUser:
+      title: StackAggregatorResultForRegisteredUser
+      allOf:
+      - $ref: '#/components/schemas/StackAggregatorResult'      
+      - type: object
+        required:
+          - analyzed_direct_dependencies
+          - vulnerable_transitives
+        properties:
+          analyzed_direct_dependencies:
+            type: array
+            items:
+              $ref: '#/components/schemas/PackageDetailsForRegisteredUser'
+            description: |
+              All direct dependencies details regardless of it's vulnerability status
+          vulnerable_transitives:
+            type: array
+            items:
+              allOf:
+                - type: object
+                  properties:
+                    dependents:
+                      type: array
+                      description: Lists all direct dependencies of this package
+                      items:
+                        $ref: '#/components/schemas/Package'
+                      example:
+                        - name: jinja2
+                          version: 2.5.2
+                        - name: django-orm
+                - $ref: '#/components/schemas/PackageDetailsForRegisteredUser'
+            description: |
+              Only Transitives which has vulnerability associated with it.
+      description: Stack analyses result for registered user
+      
+    StackAggregatorResultForFreeTier:
+      title: StackAggregatorResultForFreeTier
+      allOf:
+      - $ref: '#/components/schemas/StackAggregatorResult'
+      - type: object
+        required:
+          - registration_link
+          - analyzed_direct_dependencies
+          - vulnerable_transitives
+        properties:
+          registration_link:
+            type: string
+            format: url
+          analyzed_direct_dependencies:
+            type: array
+            items:
+              $ref: '#/components/schemas/PackageDetailsForFreeTier'
+            description: |
+              All direct dependencies details regardless of it's vulnerability status
+          vulnerable_transitives:
```

Comment:
This should not be required.

Reply:
As discussed, now it is part of PackageDetails.